vault/changelog.release.2022-07-26.md

```diff
@@ -0,0 +1,74 @@
+---
+id: xhcrzg39oqahvb89jb2m1ur
+title: 0.105
+desc: ''
+updated: 1658762256849
+created: 1658758655969
+---
+
+Dendron 0.105 has sprouted  🌱
+July 26, 2022
+
+**New Display Options in Graph Panel:** You can now toggle the display of linked and hierarchal edges in graph panel.
+
+- More information: [[Toggle Edges|dendron://dendron.dendron-site/dendron.topic.sidebar.graph-panel#toggle-edges]]
```

Comment:
i made some updates on this in dev, can you update the link to [[Toggle Edges Links|dendron://dendron.dendron-site/dendron.topic.sidebar.graph-panel#toggle-edges-links]]?